ui/lib/ldap/addon/components/page/roles.hbs

```diff
@@ -113,5 +119,15 @@
         </Item.menu>
       </ListItem>
     {{/each}}
+
```

Comment:
When implementing this, I needed to look at the hds code a lot. Was that just me or did you find the documentation sufficient? I was trying to figure out if I wanted to show the size selector, but it was too much for KV because we either use a default pageSize or override if they set the value on a config (which is crazy to set a page size value on a config, IMO).

Reply:
I pretty much copied what you had in KV and then asked whether we allow the user to select size and the response I got was not right now so it made it easier to just hide the selector.

Reply:
Ha! Okay. The size selector option is pretty rad, but it's much easier to implement with our setup without it.